firebase-abt/src/test/java/com/google/firebase/abt/FirebaseABTWithoutAnalyticsTest.java

```diff
@@ -96,6 +96,49 @@ public void replaceAllExperimentsWithoutAnalytics_sendsValidExperimentList_throw
     assertThat(actualException).hasMessageThat().contains("Analytics");
   }
 
+  @Test
+  public void reportRunningExperimentsWithoutAnalytics_throwsAbtException() {
+    List<AbtExperimentInfo> experimentInfos = new ArrayList<>();
```

Comment:
is there another test that's not just 'without analytics' we can update?

Reply:
sadly no, these are the only tests